### DISCUSSION TITLE
Unexpected behaviour

### DISCUSSION THREAD

**Discussion by Sakriel** [devbis/z03mmc#127] [Q&A]
Heyy - I wanted to upgrade my xiaomi bluetooth temp/humidity sensor. I reached state where i uploaded the [ATC_ota_40000.bin](https://github.com/devbis/z03mmc/releases/download/1.1.0/ATC_ota_40000.bin) and it worked fine. Now i can modify config etc. The next step should be uploading [z03mmc.bin](https://github.com/devbis/z03mmc/releases/download/1.1.0/z03mmc.bin) BUT! when i click start flashing it hangs on: "Status: Clear Ext.OTA region (129kib)..." Idk what should i do - can you help me please?

**Comment by rdkleine** [comment]
I have multiple sensors. First one flashed without issues and works good. Second one I get the same problem.
No idea as of now. Moving on to nr 3.

```
15:16:43: Searching for devices
15:17:02: Get Advertising MAC (Web Experimental Features!)
15:17:02: TypeError: device.watchAdvertisements is not a function
15:17:02: Reconnect 1 from 5
15:17:16: Hardware Revision String: B1.7
15:17:16: Software Revision String: V9.9
15:17:16: Firmware Revision String: github.com/devbis
15:17:16: Detected custom Firmware
15:17:16: Hardware Version: LYWSD03MMC B1.7, Software Version: 9.9, Sensor: SHT4x
15:17:16: Custom config HEX string: 55852000002804a93131043c
15:17:24: Start DFU
15:17:28: Clear Ext.OTA region (129kib)...
17:18:33: Disconnected.
```

**Comment by AlexFoks2103** [comment]
the same thing, has there been a solution?
